akka-http-jwt/src/main/scala/akka/http/jwt/scaladsl/server/directives/JwtDirectives.scala

```diff
@@ -13,18 +13,35 @@ import akka.http.scaladsl.server.directives.Credentials
 import akka.http.scaladsl.server.directives.SecurityDirectives.{ Authenticator, authenticateOAuth2 }
 import spray.json.JsObject
 
+/**
+ * JwtDirectives provides utilities to easily assert and extract claims from a JSON Web Token (JWT).
```

Comment:
```suggestion
 * JwtDirectives provides utilities to assert and extract claims from a JSON Web Token (JWT).
```
We try to avoid "easily" and "simply" in docs since it can feel condescending for a struggling user.